src/osd/PG.cc

```diff
@@ -754,9 +754,7 @@ void PG::generate_past_intervals()
   }
 
   // Verify same_interval_since is correct
```

Comment:
Delete this comment.  Could say

// PG import needs recalculated same_interval_since